sdk/resourcemanager/Azure.ResourceManager/src/ArmClient.cs

```diff
@@ -25,6 +25,7 @@ public class ArmClient
         /// The base URI of the service.
         /// </summary>
         internal const string DefaultUri = "https://management.azure.com";
+        internal const string Scope = "https://management.core.windows.net/.default";
```

Comment:
HI @fengzhou-msft do you have idea how non-commercial clouds are supported in .net sdk? given the hardcoded url? thanks.

Reply:
This is the default public cloud value. This PR allows the Scope to be specified via options to override this one.